packages/grafana-data/src/themes/zIndex.ts

```diff
@@ -9,7 +9,6 @@ export const zIndex = {
   tooltip: 1040,
```

Comment:
Is `tooltip` now obsolete? 🤔 would it make more sense push tooltips up to 1052 or something / rearrange all of the other properties up (i.e. modalBackdrop becomes 1040, modal: 1050, portal: 1051, tooltips: 1052)

Reply:
I think that's a bigger conversation and a good question.

All I can say is that within `TooltipPlugin2` it is no longer required, but I can't comment on the ~12 other cases where it is still being used, and how other tooltips might interact with modals (either by being visible from "below" the modal or being hidden under the modal as in this case).

Reply:
> Is tooltip now obsolete? 🤔

code search shows that `zIndex.tooltip` is still used in other UI components (maybe for infotip / toggletip)?